addons/web/static/tests/core/popover/popover_tests.js

```diff
@@ -264,3 +264,37 @@ QUnit.test("within iframe", async (assert) => {
     assert.strictEqual(popoverBox.top, expectedTop);
     assert.strictEqual(popoverBox.left, expectedLeft);
 });
+
+QUnit.test("popover fixed position", async (assert) => {
+    const container = document.createElement("div");
+    container.id = "container";
+    container.style.backgroundColor = "pink";
+    container.style.height = "450px";
+    container.style.width = "450px";
+    container.style.display = "flex";
+    container.style.alignItems = "center";
+    container.style.justifyContent = "center";
+    popoverTarget.style.backgroundColor = "yellow";
+    popoverTarget.style.height = "50px";
+    container.appendChild(popoverTarget);
+    fixture.appendChild(container);
+
+    const TestPopover = class extends Popover {
+        onPositioned(el, { direction, variant }) {
+            assert.step("onPositioned");
+        }
+    };
+    await mount(TestPopover, fixture, {
+        props: { target: container, position: "bottom-fit", fixedPosition: true },
+    });
```

Comment:
(I assume they already have tests to make sure the popover is correctly repositioned when your new config is not specified?)

Reply:
yes, above mine :)